Sts1CobcSw/Hal/Spi.hpp

```diff
@@ -10,6 +10,25 @@
 
 namespace sts1cobcsw::hal
 {
+class Spi
+{
+private:
+    RODOS::HAL_SPI spi;
+    RODOS::CommBuffer<std::int64_t> transferEnd;
```

Comment:
List private members after the public members. The public interface is what's interesting to most people.